sound/soc/intel/boards/sdw_rt700.c

```diff
@@ -105,6 +106,53 @@ static int card_late_probe(struct snd_soc_card *card)
 }
 #endif
 
+static struct snd_soc_jack_pin sdw_jack_pins[] = {
+	{
+		.pin    = "Headphones",
+		.mask   = SND_JACK_HEADPHONE,
```

Comment:
Minor note: in most drivers these are labelled as "Headphone Jack" and "Mic Jack". Of course this is aligning to existing code in the machine driver, so maybe not for this patch.

Reply:
Agree.

Reply:
@bardliao did you update this? I see no change?

Reply:
> @bardliao did you update this? I see no change?

No, should I do it?

Reply:
let's do it in a follow-up, if needed